oxidation/libparsec/crates/types/tests/mod.rs

```diff
@@ -1,5 +1,8 @@
 // Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 (eventually AGPL-3.0) 2016-present Scille SAS
 
+// Functions using rstest parametrize ignores `#[warn(clippy::too_many_arguments)]`
+// decorator, so we must do global ignore instead :(
+#![allow(clippy::too_many_arguments)]
```

Comment:
Maybe it's the indication that we need to define/use a struct :shrug:

Reply:
Yes but out of scope

Reply:
Open an issue

Reply:
https://github.com/Scille/parsec-cloud/issues/4371